One of the integration tests seems to fail intermittently

I've observed an intermittent failure in ```devel.ci.integration.tests.test_bodhi_cli.test_updates_download```, in #2754 and #2831. For example:

https://ci.centos.org/job/bodhi-pipeline/job/PR-2831/1/testReport/junit/devel.ci.integration.tests/test_bodhi_cli/test_updates_download/

In both cases, the expected output was not found in the mocked Koji output:

```
AssertionError: assert 'Downloading packages from setroubleshoot-plugins-3.3.10-1.fc28' in 'Downloading packages from git-2.20.0-1.fc29\nTESTING CALL /usr/bin/koji download-build git-2.20.0-1.fc29\nDownloading...d-build setroubleshoot-3.3.19-1.fc28\nTESTING CALL /usr/bin/koji download-build setroubleshoot-plugins-3.3.10-1.fc28\n'  +  where 'Downloading packages from setroubleshoot-plugins-3.3.10-1.fc28' = <built-in method format of str object at 0x7f2fa0ab8c60>('setroubleshoot-plugins-3.3.10-1.fc28')  +    where <built-in method format of str object at 0x7f2fa0ab8c60> = 'Downloading packages from {}'.format  +  and   'Downloading packages from git-2.20.0-1.fc29\nTESTING CALL /usr/bin/koji download-build git-2.20.0-1.fc29\nDownloading...d-build setroubleshoot-3.3.19-1.fc28\nTESTING CALL /usr/bin/koji download-build setroubleshoot-plugins-3.3.10-1.fc28\n' = Munch({'exit_code': 0, 'output': 'Downloading packages from git-2.20.0-1.fc29\nTESTING CALL /usr/bin/koji download-bui...build setroubleshoot-3.3.19-1.fc28\nTESTING CALL /usr/bin/koji download-build setroubleshoot-plugins-3.3.10-1.fc28\n'}).output
```

--- a/devel/ci/integration/tests/test_bodhi_cli.py
+++ b/devel/ci/integration/tests/test_bodhi_cli.py
@@ -289,7 +289,7 @@ def test_updates_download(bodhi_container, db_container):
     # Fetch the last updates from the DB
     builds = []
     db_ip = db_container.get_IPv4s()[0]
-    query_updates = "SELECT id, alias FROM updates ORDER BY date_submitted DESC LIMIT 3"
+    query_updates = "SELECT id, alias, title FROM updates ORDER BY date_submitted DESC LIMIT 3"
     query_builds = "SELECT nvr FROM builds WHERE update_id = %s ORDER BY nvr"
     conn = psycopg2.connect("dbname=bodhi2 user=postgres host={}".format(db_ip))
     with conn:
@@ -311,6 +311,7 @@ def test_updates_download(bodhi_container, db_container):
     with replace_file(bodhi_container, "/usr/bin/koji", koji_mock):
         result = _run_cli(bodhi_container, cmd)
     assert result.exit_code == 0
+    for update in updates:
+        assert "Downloading packages from {}".format(update['title']) in result.output
     for build_id in builds:
-        assert "Downloading packages from {}".format(build_id) in result.output
         assert "TESTING CALL /usr/bin/koji download-build {}".format(build_id) in result.output

